### ISSUE TITLE
Appears to be broken on chrome os 99

### ISSUE THREAD

**Issue by linusrg1** [CerisWhite/CrAnberry#22]
I can install it just fine, but then android apps fail to launch. Unlike chrome os 98 where everything seemed to work just fine.

**Comment by CerisWhite** [comment]
Yes, I've been told by a few people that ChromeOS has recently updated to using Android 10, which makes this script obsolete. I do not have any ChromeOS devices that can use v99, so I can't fix this right now.

Once again, if anyone wants to work with me to fix this, I will try everything I can.

**Comment by linusrg1** [comment]
> Yes, I've been told by a few people that ChromeOS has recently updated to using Android 10, which makes this script obsolete. I do not have any ChromeOS devices that can use v99, so I can't fix this right now.
> 
> Once again, if anyone wants to work with me to fix this, I will try everything I can.

mine is still stuck on pie for some reason

**Comment by schrega** [comment]
Version 103.0.5060.114  here. I was having the same problem with the script. Today I managed to get it rooted doing 2 different steps:
1- instead disabling roorfs writing protection with the given script, I used the one that crosh gave me:
 sudo /usr/share/vboot/bin/make_dev_ssd.sh --remove_rootfs_verification --partitions 2

2- I downloaded cranberry.sh first and moved it as sugested, instead running with CUrl. 

After that the script run with just some errors creating folders that already existed, supersu download ran twice, but for everything else no erros.

Titanium backup reporting root access, backin up and restoring properly, *BUT* I dont have supersu app anywhere.

**Comment by VenomousSteam81** [comment]
Hmmm... Maybe rooting does still work.... I'm trying it now and I will post when it gets finished!

**Comment by VenomousSteam81** [comment]
Holy shit it works. Android 9

**Comment by CerisWhite** [comment]
@VenomousSteam81 Good to know. Thank you for reporting back! Please let me know what version of ChromeOS you're using and what device as well.

**Comment by VenomousSteam81** [comment]
Version: 104.0.5112.83 (Official Build) (32-bit)
Chromebook: Lenovo 100e 2nd Gen MTK

**Comment by androiduser505** [comment]
> Version: 104.0.5112.83 (Official Build) (32-bit) Chromebook: Lenovo 100e 2nd Gen MTK
> 
> @CerisWhite

um dude is that android 9?

**Comment by midozalouk** [comment]
Any news ???
